std/src/main/scala/algebra/std/set.scala

```diff
@@ -6,12 +6,15 @@ import algebra.ring.Semiring
 package object set extends SetInstances
 
 trait SetInstances {
-  implicit def setEq[A]: Eq[Set[A]] = new SetEq[A]
+  implicit def setPartialOrder[A]: PartialOrder[Set[A]] = new SetPartialOrder[A]
   implicit def setSemiring[A] = new SetSemiring[A]
 }
 
-class SetEq[A] extends Eq[Set[A]] {
-  def eqv(x: Set[A], y: Set[A]): Boolean = x == y
+class SetPartialOrder[A] extends PartialOrder[Set[A]] {
+  def partialCompare(x: Set[A], y: Set[A]): Double =
+    if (x == y) 0.0
+    else if (x.subsetOf(y)) -1.0
+    else 1.0
```

Comment:
`y.subsetOf(x)` makes sense to me, but the `else Double.NaN` would break `PartialOrder` laws, where

`x > y --> y < x`

Looking at: https://github.com/non/algebra/blob/master/core/src/main/scala/algebra/PartialOrder.scala#L111

Given sets `x = Set(0, 1), y = Set(0)`, `x > y` yields `true` but `y < x` yields `Double.NaN`.